components/eventing-controller/pkg/handlers/utils.go

```diff
@@ -0,0 +1,91 @@
+package handlers
+
+import (
+	"fmt"
+	"strings"
+
+	eventingv1alpha1 "github.com/kyma-project/kyma/components/eventing-controller/api/v1alpha1"
+
+	"github.com/kyma-project/kyma/components/eventing-controller/pkg/ems/api/events/types"
+	"github.com/mitchellh/hashstructure"
+)
+
+func getHash(subscription *types.Subscription) (int64, error) {
+	if hash, err := hashstructure.Hash(subscription, nil); err != nil {
```

Comment:
Why not use the [hash/fnv](https://golang.org/pkg/hash/fnv/) from go-sdk?

See some reference implementation by the Kubenetes controller https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/controller_utils.go#L1130-L1145.

Reply:
hashstructure.Hash() is a "deep hash" function

Reply:
It will be changed in https://github.com/kyma-project/kyma/issues/9766